.rubocop.yml

```diff
@@ -55,3 +52,18 @@ Style/StringLiterals:
 
 Style/WordArray:
   EnforcedStyle: brackets
+
+Style/StringLiterals:
+  Enabled: false
+
+Metrics/ClassLength:
+  Enabled: false
+
+Metrics/ModuleLength:
+  Enabled: false
+
+Metrics/MethodLength:
```

Comment:
Not sure how I missed that. You're absolutely right. All these `Enabled: false` aren't necessary.